tabbycat/notifications/views.py

```diff
@@ -343,21 +349,21 @@ def form_valid(self, form):
 
 class TournamentTemplateEmailCreateView(TemplateEmailCreateView):
 
-    def get_default_send_queryset(self):
+    def get_default_send_queryset(self) -> 'QuerySet[Person]':
         return super().get_default_send_queryset().exclude(
             sentmessage__notification__event=self.event, sentmessage__notification__tournament=self.tournament)
 
-    def get_extra(self):
+    def get_extra(self) -> Dict[str, int]:
```

Comment:
As used elsewhere, think `Any` should be used. It's odd to have an extending method (`EmailRandomisedUrlsView.get_extra()`) with a different return type.

Also for `RoundTemplateEmailCreateView`.